tools/functional-tester/etcd-tester/main.go

```diff
@@ -111,17 +117,24 @@ func main() {
 			schedule[i] = failures[caseNum]
 		}
 	}
-
 	t := &tester{
 		failures: schedule,
 		cluster:  c,
 		limit:    *limit,
-		checker:  newNoChecker(),
 	}
 
+	checkers := make([]Checker, 0)
+
 	if *consistencyCheck && !c.v2Only {
-		t.checker = newHashChecker(t)
+		checkers = append(checkers, newHashChecker(t))
+	} else {
+		checkers = append(checkers, newNoChecker())
 	}
+	lc := newLeaseChecker(c.LeaseStressers)
```

Comment:
There's no need to track leaseStressers throughout the tester if there's something like:
```go
var leaseStressers []*leaseStresser
f := func(ls *leaseStresser) { leaseStressers = append(leaseStressers, ls) }
// pass f to the lease stresser builder; call f(ls) when building a new lease stresser
lc := newLeaseChecker(leaseStressers)
```